crates/oxc_linter/src/service/runtime.rs

```diff
@@ -274,6 +274,17 @@ impl Runtime {
     ) {
         if self.resolver.is_none() {
             self.paths.par_iter().for_each(|path| {
+                if self
```

Comment:
I think we may still want to process it here, so that we can add the modules info to the module graph (regardless of whether it's actaully going to be linted.)

Reply:
🤦 haha i should've read L 275

Reply:
> Will fix it in line 380 by moving it before the module_to_lint.push

yes this change seems good to me, thanks

Reply:
With your request of the real linted file numbers. I am moving the scope to the tools (oxlint / language server).
The server does not need to know the numbers of files.